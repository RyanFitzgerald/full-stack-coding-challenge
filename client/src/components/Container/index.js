import styled from 'styled-components';

const Container = styled.div`
  margin: 0 auto;
  max-width: 960px;
  padding: 15px;

  &:after {
    content: '';
    display: table;
    clear: both;
  }
`;

export default Container;