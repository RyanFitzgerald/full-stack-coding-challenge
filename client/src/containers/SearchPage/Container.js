import styled from 'styled-components';
import OriginalContainer from '../../components/Container';


const Container = styled(OriginalContainer)`
  padding: 50px 15px;
`;

export default Container;