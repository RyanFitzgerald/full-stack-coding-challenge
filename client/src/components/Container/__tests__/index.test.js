import React from 'react';

import Container from '../index';

describe('<Container />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer.create(
      <Container>Test</Container>
    ).toJSON();
    
    expect(renderedComponent).toMatchSnapshot();
  });

  it('should render the correct element', () => {
    const renderedComponent = shallow(
      <Container/>
    );
    
    expect(renderedComponent.type()).toEqual('div');
  });

  it('should render its content', () => {
    const children = 'Text';
    const renderedComponent = shallow(
      <Container>{children}</Container>
    );
    
    expect(renderedComponent.contains(children)).toBe(true);
  });
});