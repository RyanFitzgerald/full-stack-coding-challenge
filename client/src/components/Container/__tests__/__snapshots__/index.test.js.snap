// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should match the snapshot 1`] = `
.c0 {
  margin: 0 auto;
  max-width: 960px;
  padding: 15px;
}

.c0:after {
  content: '';
  display: table;
  clear: both;
}

<div
  className="c0"
>
  Test
</div>
`;
